websites/A/AnimeSaturn/dist/metadata.json

```diff
@@ -1,5 +1,9 @@
 {
   "author": {
+    "name": "Dark_Ville",
+    "id": "638080361179512853"
+  },
+  "contributor": {
     "name": "Tellen",
     "id": "535214819520020510"
   },
```

Comment:
contributors should be in an array
and why even change of author?

Reply:
Taking over cause not maintained (issues open since Apr 8 2021)

Reply:
Did you contact the maintainer tho?